app/views/layouts/application.html.slim

```diff
@@ -13,6 +13,9 @@ html lang="#{I18n.locale}" class='no-js'
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
+    <!--[if IE 8]>
+      <script src='https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js'></script>
```

Comment:
Correct, IE basically doesn't support CSP headers at all (edge supports a tiny subset), so we don't need to add it.